chia/plot_sync/receiver.py

```diff
@@ -357,6 +375,8 @@ def to_dict(self, counts_only: bool = False) -> Dict[str, Any]:
             "no_key_filenames": get_list_or_len(self._keys_missing, counts_only),
             "duplicates": get_list_or_len(self._duplicates, counts_only),
             "total_plot_size": self._total_plot_size,
+            "total_effective_plot_size": int(self._total_effective_plot_size),
```

Comment:
Why do we need to add this here to the JSON which makes it part of the RPC? If we for some reason want to add it to RPC i think we should just apply `UI_ACTUAL_SPACE_CONSTANT_FACTOR` here instead of storing it also in `Receiver. _total_effective_plot_size`.